packages/tab-bar/index.js

```diff
@@ -11,8 +11,11 @@ export default class TabBar extends Component {
   tabList_ = [];
 
   foundation_ = null;
-  state = {
-    previousActiveIndex: -1,
+  constructor(props) {
+    super(props);
+    this.state = {
+      previousActiveIndex: props.activeIndex,
+    };
   }
 
```

Comment:
In the process of debugging the screenshot tests I found another bug: we forgot to update `state.previousActiveIndex` when `props.activeIndex` updates
```suggestion
  componentDidUpdate(prevProps) {
    if (this.props.activeIndex !== prevProps.activeIndex) {
      this.setState({previousActiveIndex: prevProps.activeIndex},
        () => this.foundation_.activateTab(this.props.activeIndex));
    }
    if (this.props.indexInView !== prevProps.indexInView) {
      this.foundation_.scrollIntoView(this.props.indexInView);
    }
  }
```